--- origsrc/gst-ffmpeg-0.10.6/autogen.sh	2009-02-05 00:17:19.003750000 -0600
+++ src/gst-ffmpeg-0.10.6/autogen.sh	2009-02-05 00:14:09.000000000 -0600
@@ -97,7 +97,7 @@ fi
 
 toplevel_check $srcfile
 
-tool_run "$aclocal" "-I m4 -I common/m4 $ACLOCAL_FLAGS"
+tool_run "$aclocal" "-I common/m4 $ACLOCAL_FLAGS"
 tool_run "$libtoolize" "--copy --force"
 tool_run "$autoheader"
 
--- origsrc/gst-ffmpeg-0.10.6/configure.ac	2009-02-05 00:17:19.003750000 -0600
+++ src/gst-ffmpeg-0.10.6/configure.ac	2009-02-05 00:16:55.488125000 -0600
@@ -265,9 +265,9 @@ else
                  -Wno-deprecated-declarations"
 
   dnl libgstffmpeg.la: libs to statically link to        
-  FFMPEG_LIBS="\$(top_builddir)/gst-libs/ext/ffmpeg/libavformat/libavformat.a \
-               \$(top_builddir)/gst-libs/ext/ffmpeg/libavcodec/libavcodec.a \
-               \$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a"
+  FFMPEG_LIBS="-Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libavformat/libavformat.a \
+               -Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libavcodec/libavcodec.a \
+               -Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a,-lz,-lbz2"
   dnl
   POSTPROC_CFLAGS="-I \$(top_srcdir)/gst-libs/ext/ffmpeg/libpostproc	\
                    -I \$(top_srcdir)/gst-libs/ext/ffmpeg/libavutil 	\
@@ -276,8 +276,8 @@ else
                    -Wno-deprecated-declarations"
 
   dnl libgstpostproc.la: libs to statically link to
-  POSTPROC_LIBS="\$(top_builddir)/gst-libs/ext/ffmpeg/libpostproc/libpostproc.a \
-  		 \$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a"
+  POSTPROC_LIBS="-Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libpostproc/libpostproc.a \
+  		 -Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a"
   				 
   dnl
   SWSCALE_CFLAGS="-I \$(top_srcdir)/gst-libs/ext/ffmpeg/libswscale	\
@@ -286,8 +286,8 @@ else
                   -Wno-deprecated-declarations"
 
   dnl libgstswscale.la: libs to statically link to
-  SWSCALE_LIBS="\$(top_builddir)/gst-libs/ext/ffmpeg/libswscale/libswscale.a \
-                \$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a"
+  SWSCALE_LIBS="-Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libswscale/libswscale.a \
+                -Wl,\$(top_builddir)/gst-libs/ext/ffmpeg/libavutil/libavutil.a"
 
   FFMPEG_SUBDIRS=gst-libs
   AC_DEFINE(HAVE_AVI_H)
--- origsrc/gst-ffmpeg-0.10.6/ext/ffmpeg/Makefile.am	2009-02-05 00:17:19.035000000 -0600
+++ src/gst-ffmpeg-0.10.6/ext/ffmpeg/Makefile.am	2009-02-05 00:14:09.000000000 -0600
@@ -19,9 +19,6 @@ libgstffmpeg_la_LIBADD = $(FFMPEG_LIBS) 
 libgstffmpeg_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS) $(DARWIN_LDFLAGS)
 libgstffmpeg_la_LIBTOOLFLAGS = --tag=disable-static
 
-if HAVE_FFMPEG_UNINSTALLED
-libgstffmpeg_la_DEPENDENCIES = $(FFMPEG_LIBS)
-endif
 
 
 noinst_HEADERS = \
--- origsrc/gst-ffmpeg-0.10.6/ext/libswscale/Makefile.am	2008-11-04 06:49:40.000000000 -0600
+++ src/gst-ffmpeg-0.10.6/ext/libswscale/Makefile.am	2009-02-05 01:53:42.738125000 -0600
@@ -11,6 +11,3 @@ libgstffmpegscale_la_LIBADD = $(SWSCALE_
 libgstffmpegscale_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS) $(DARWIN_LDFLAGS)
 libgstffmpegscale_la_LIBTOOLFLAGS = --tag=disable-static
 
-if HAVE_FFMPEG_UNINSTALLED
-libgstffmpegscale_la_DEPENDENCIES = $(SWSCALE_LIBS)
-endif
